admin-dev/themes/default/template/layout-ajax.tpl

```diff
@@ -79,13 +79,13 @@
 		<div class="alert alert-warning">
 			<button type="button" class="close" data-dismiss="alert">&times;</button>
 			{if count($warnings) > 1}
-				{l s='There are %d warnings.' sprintf=count($warnings)}
+				{l s='There are %d warnings.' sprintf=count($warnings) d='Admin.Notifications.Error'}
 				<span style="margin-left:20px;" id="labelSeeMore">
 					<a id="linkSeeMore" href="#" style="text-decoration:underline">{l s='Click here to see more'}</a>
```

Comment:
What about _See more_ in _Admin.Actions_?

Reply:
We might have forgotten this suggestion. :-)

Reply:
Apply it to line 84, instead of _Click here to see more_, please > _See more_ in _Admin.Actions_